### PR TITLE
[POU-75] Clean project structure

### PR DESCRIPTION
**Pull request by crimson-med**
Item POU-75

### CODE REVIEW

File: src/components/organisms/Header/__snapshots__/Header.spec.tsx.snap [hunk 1]
```diff
@@ -6,6 +6,7 @@ exports[`renders correctly 1`] = `
     class="css-0"
   >
     <img
+      alt="CAD Banner"
```
Comment (morphet81) [inline src/components/organisms/Header/__snapshots__/Header.spec.tsx.snap:9]: Alt should be localized

File: src/components/organisms/Header/Header.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import headerBanner from './../../../assets/header.jpg'
 export const Header = (): React.JSX.Element => {
   return (
     <Box>
-      <Image src={headerBanner.src} />
+      <Image alt="CAD Banner" src={headerBanner.src} />
```
Comment (morphet81) [inline src/components/organisms/Header/Header.tsx:7]: Localize alts

**Review by morphet81 — CHANGES_REQUESTED** [review]

File: src/components/organisms/Header/__snapshots__/Header.spec.tsx.snap [hunk 3]
```diff
@@ -6,6 +6,7 @@ exports[`renders correctly 1`] = `
     class="css-0"
   >
     <img
+      alt="images.bannerImage"
```
Comment (morphet81) [inline src/components/organisms/Header/__snapshots__/Header.spec.tsx.snap:9]: Are you sure this works withou using the `t` function?

**Review by morphet81 — APPROVED** [review]

File: src/components/organisms/Header/__snapshots__/Header.spec.tsx.snap (hunk 3 above)
Comment (crimson-med) [reply src/components/organisms/Header/__snapshots__/Header.spec.tsx.snap:9]: Yes as this is the generated snapshot test. The test executes with no issues

**Review by morphet81 — APPROVED** [review]

**Pull request merged**